examples/extract_results.py

```diff
@@ -34,10 +36,27 @@ def main_extract(content):
         secondL2=float(secondOrder.split(',')[1])
 
         results.append(["{0:0.2e}".format(i) for i in [firstLinf,firstL2,secondLinf,secondL2]])
+
```

Comment:
Not a problem with this specific PR, since it's internal, but in general when you send out PRs to other projects owned by people you don't know, you don't want to include into the PR things unrelated to the purpose of the PR, like adding new lines. In general, you want the PR diffs to be as small as possible as to make it easier for developers to see what you changed and reason about the changes.